addons/account/models/partner.py

```diff
@@ -284,6 +295,10 @@ def ranking_function(fpos):
             default=(self.env['account.fiscal.position'], False)
         )[0]
 
+    def action_open_related_taxes(self):
+        self.ensure_one()
```

Comment:
Who care about the ensure_one?
If someone wants to use this action in multi, why block them?